geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+images and its application to structural study of carbon tetrachloride. J Mol Struct 978 (1-3):61-66
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C    Cl 1.768(2)
+Cl...Cl 2.885(1)
+
+# Bond lengths: rg     (angstroms)
+
+C    Cl 1.770(2)
+Cl...Cl 2.887(1)
+
+# Bond lengths: rα     (angstroms)
+
+C    Cl 1.767(2)
+Cl...Cl 2.886(1)
+
+# Bond lengths: rh1    (angstroms)
+
+C    Cl 1.769(2)
+Cl...Cl 2.888(1)
+
+NAME = CF2O 2012
+# Point Group: C2v
+# Shape: Trigonal planar
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Demaison J, Császár AG (2012) Equilibrium CO bond lengths. J Mol Struct 1023:7-14
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C = O   1.1699(4)
+C  F    1.3100(2)
+F  C  F 107.80(3)
+
+NAME = CFeN
+# Point Group: Coov
+# Shape: Linear
+# Bond lengths: r0     (angstroms)
+# Source Paper: Flory MA, Ziurys LM (2011) Millimeter-wave rotational spectroscopy of FeCN (X4
+Δi) and FeNC (X6Δi): Determining the lowest energy isomer. J Chem Phys 135(18):184303/1-184303/11
+doi:10.1063/1.3653809
+[http://dx.doi.org/10.1063/1.3683221]
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Fe   C 1.9244(8)
+C  ≡ N 1.157(1)
+
+# Bond lengths: rs     (angstroms)
+
+Fe   C 1.9355(68)
+C  ≡ N 1.146(23)
+
+# Bond lengths: rm^(1)     (angstroms)
+
+Fe   C 1.917(12)
+C  ≡ N 1.128(21)
+
+NAME = CFeO
+# Point Group: Coov
+# Shape: Linear
+# Bond lengths: re     (angstroms)
+# Source Paper: Tanaka K, Nakamura M, Shirasaka M, Sakamoto A, Harada K, Tanaka T (2015) Millimeter-wave spectroscopy
+of the FeCO radical in the ν2 and ν3 vibrationally excited states. J Chem Phys 143(1):014303/1-014303/10
+[http://dx.doi.org/10.1063/1.4923215]
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+Fe   C 1.725
+C  ≡ O 1.1587
+
+# Bond length C ≡ O 1.1587 asummed according to ab initio calculations
+
+NAME = CHBr3 2009
+# Point Group: C3v
+# Shape: Tetrahedral
+# Bond lengths: rz     (angstroms)
+# Bond angles:  thetaz (degrees)
+# Source Paper: Kisiel Z, Kraśnicki A, Pszczółkowski L, Shipman ST, Alvarez-Valtierra L, Pate BH (2009) Assignment and
+analysis of the rotational spectrum of bromoform enabled by broadband FTMW spectroscopy. J Mol Spectrosc
+257(2):177-186
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C  H     1.0831(32)
+C  Br    1.9260(3)
+H  C  Br 107.622(27)
+Br C  Br 111.255(25)
+
+NAME = CHClF2 2014
+# Point Group: Cs
+# Shape: Tetrahedral
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+# Source Paper: Vogt N, Demaison J, Rudolph HD (2014) Accurate equilibrium structures of fluoro- and chloroderivatives of
+methane. Mol Phys 112(22):2873-2883
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C H     1.0850(11)
+C F     1.3363(5)
+C Cl    1.7560(9)
+H C  F  109.97(4)
+H C  Cl 109.60(6)
+F C  Cl 109.62(4)
+F C  F  108.06(6)
+
+NAME = CHNS
+# Point Group: Coov
```

Comment:
How is this distinguished from the other two CHNS structures? Unfortunately all three are published in 2016. The Coov point group distinguishes them, but the other two are both Cs so we might need to think of a different way to distinguish them.